playwright/mocks/generateTypes.ts

```diff
@@ -0,0 +1,48 @@
+import { CustomType } from "@prismicio/types-internal/lib/customtypes";
+
+import { Library } from "./generateLibraries";
+import { generateRandomId } from "../utils";
+
+type GenerateTypesArgs = {
+  nbTypes: number;
```

Comment:
Same comment as above: is there a clearer name than "nb"?

Reply:
`typesCount`, thanks, you're right